### ISSUE TITLE
Documentation/'credentials-updated'

### ISSUE THREAD

**Issue by jaovito** [adiwajshing/Baileys#471]
In documentation have the property 'credentials-updated', but does not exists in the lib.

**Comment by FRM-Developer** [comment]
after I scanned the qr code, the credentials were not saved. even though it is correct, nothing is missing

**Comment by adiwajshing** [comment]
@jaovito the method was removed quite a while back as it was redundant. Please use `open` instead

**Comment by h1t35h** [comment]
Lord! I have been fiddling around the code base for 1 hour just to open issues and find this. Should I send a pull request to update the README for this one?

**Comment by adiwajshing** [comment]
Yes please, I would very grateful. It's my bad, I thought I had updated all occurrences.
